src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  position: relative;
+}
+
+.search__input {
+  display: block;
+  font-size: inherit;
+  line-height: inherit;
```

Comment:
it's a redundant rule
```suggestion

```

Reply:
> it's a redundant rule

"line-height: inherit" This rule has been removed. Thanks